datadog_checks_dependency_provider/datadog_checks/datadog_checks_dependency_provider/data/conf.yaml.example

```diff
@@ -0,0 +1,6 @@
+init_config: {}
+instances:
+    ## @param required - boolean - required
```

Comment:
A bit confused here. Do all integrations require at least one config parameter so this is a fake placeholder?

Reply:
Yes, by the legacy config validator